parity-crypto/README.md

```diff
@@ -3,3 +3,8 @@
 General cryptographic utilities for Ethereum.
 
 By default, this library is compiled with the `secp256k1` feature, which provides ECDH and ECIES capability on that curve. It can be compiled without to avoid a dependency on the `libsecp256k1` library.
+
+
+## Changelog
+
+The 0.4 release removes the dependency on `ring` and replaces it with prue-rust alternatives. As a consequence of this, AES GCM support has been removed. `subtle` replaces the `constant_time_eq` crate for constant time equality testing.
```

Comment:
Actually `subtle` replaces `ring::constant_time::verify_slices_are_equal`, but I'm not sure it's worth mentioning, since it doesn't affect the public API.

Reply:
You're right, 0.3.1 used `ring. Fixing the wording.